### PR TITLE
Added I press Y key X times. Added e2e test

### PR DESCRIPTION
**Pull request by eugs**

### CODE REVIEW

**Review by AlexGalichenko — APPROVED** [review]
Looks good to me.

File: src/actions.ts [hunk 1]
```diff
@@ -158,6 +158,18 @@ When('I press {string} key', async function (key: string) {
...
 });
 
+/**
+ * Press button given number of times
+ * @param {string} key - key to press
+ * @param {number} num - number of times
+ * @example I press 'Enter' key 5 times
+ */
+When('I press {string} key {int} times', async function (key: string, num: number) {
+    for (let i = 0; i < num; i++) {
```
Comment (nomid0612) [inline src/actions.ts:168]: type for i is missed

File: test-e2e/apps/actions.html [hunk 2]
```diff
@@ -28,6 +32,8 @@
     const buttons = document.querySelectorAll('.button');
     const body = document.body;
     const select = document.querySelector('#select');
+    const pressCounter = document.querySelector('#pressCounter');
+    let presses = 0;
```
Comment (nomid0612) [inline test-e2e/apps/actions.html:36]: type for presses is missed

Comment (AlexGalichenko) [reply test-e2e/apps/actions.html:36]: I think here typescript is not supported, it's <script> tag inside html

File: src/actions.ts (hunk 1 above)
Comment (eugs) [reply src/actions.ts]: Added type

File: src/actions.ts [hunk 3]
```diff
@@ -158,6 +158,18 @@ When('I press {string} key', async function (key: string) {
     await browser.keys(key);
 });
 
+/**
+ * Press button given number of times
+ * @param {string} key - key to press
+ * @param {number} num - number of times
+ * @example I press 'Enter' key 5 times
+ */
+When('I press {string} key {int} times', async function (key: string, num: number) {
```
[the remark this answers is not in the document]
Comment (eugs) [reply src/actions.ts]: Added postfix

**Review by KirillBogdanets — APPROVED** [review]

**Pull request merged**